### PR TITLE
fix pg migrations

### PR DESCRIPTION
**Pull request by lastbyte32**
use
SQLALCHEMY_DATABASE_URL=**postgresql**+**psycopg2**://${DB_USER}:${DB_PASSWORD}@db/${DB_NAME}
<details>
<summary>Console output errors</summary>
<code >
INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
  [ 2](#L2)	INFO  [alembic.runtime.migration] Will assume transactional DDL.
  [ 3](#L3)	INFO  [alembic.runtime.migration] Running upgrade  -> 94a5cc12c0d6, init user table
  [ 4](#L4)	INFO  [alembic.runtime.migration] Running upgrade 94a5cc12c0d6 -> 3cf36a5fde73, init system table
  [ 5](#L5)	INFO  [alembic.runtime.migration] Running upgrade 3cf36a5fde73 -> 9d5a518ae432, init jwt table
  [ 6](#L6)	INFO  [alembic.runtime.migration] Running upgrade 9d5a518ae432 -> 9b60be6cd0a2, add created_at field to users table
  [ 7](#L7)	INFO  [alembic.runtime.migration] Running upgrade 9b60be6cd0a2 -> 8e849e06f131, proxy table
  [ 8](#L8)	INFO  [alembic.runtime.migration] Running upgrade 8e849e06f131 -> 671621870b02, init admin
  [ 9](#L9)	INFO  [alembic.runtime.migration] Running upgrade 671621870b02 -> e91236993f1a, inbounds table & excluded_inbounds
 [ 10](#L10)	INFO  [alembic.runtime.migration] Running upgrade e91236993f1a -> b3378dc6de01, hosts
 [ 11](#L11)	INFO  [alembic.runtime.migration] Running upgrade 671621870b02 -> d02dcfbf1517, add UserUsageResetLogs Model and data_limit_strategy field to user
 [ 12](#L12)	Traceback (most recent call last):
 [ 13](#L13)	  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1900, in _execute_context
 [ 14](#L14)	    self.dialect.do_execute(
 [ 15](#L15)	  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/engine/default.py", line 736, in do_execute
 [ 16](#L16)	    cursor.execute(statement, parameters)
 [ 17](#L17)	psycopg2.errors.UndefinedObject: type "userdatalimitresetstrategy" does not exist
 [ 18](#L18)	LINE 1: ... TABLE users ADD COLUMN data_limit_reset_strategy userdatali...
 [ 19](#L19)	                                                             ^
 [ 20](#L20)	
 [ 21](#L21)	
 [ 22](#L22)	The above exception was the direct cause of the following exception:
 [ 23](#L23)	
 [ 24](#L24)	Traceback (most recent call last):
 [ 25](#L25)	  File "/usr/local/bin/alembic", line 8, in <module>
 [ 26](#L26)	    sys.exit(main())
 [ 27](#L27)	  File "/usr/local/lib/python3.10/site-packages/alembic/config.py", line 590, in main
 [ 28](#L28)	    CommandLine(prog=prog).main(argv=argv)
 [ 29](#L29)	  File "/usr/local/lib/python3.10/site-packages/alembic/config.py", line 584, in main
 [ 30](#L30)	    self.run_cmd(cfg, options)
 [ 31](#L31)	  File "/usr/local/lib/python3.10/site-packages/alembic/config.py", line 561, in run_cmd
 [ 32](#L32)	    fn(
 [ 33](#L33)	  File "/usr/local/lib/python3.10/site-packages/alembic/command.py", line 322, in upgrade
 [ 34](#L34)	    script.run_env()
 [ 35](#L35)	  File "/usr/local/lib/python3.10/site-packages/alembic/script/base.py", line 569, in run_env
 [ 36](#L36)	    util.load_python_file(self.dir, "env.py")
 [ 37](#L37)	  File "/usr/local/lib/python3.10/site-packages/alembic/util/pyfiles.py", line 94, in load_python_file
 [ 38](#L38)	    module = load_module_py(module_id, path)
 [ 39](#L39)	  File "/usr/local/lib/python3.10/site-packages/alembic/util/pyfiles.py", line 110, in load_module_py
 [ 40](#L40)	    spec.loader.exec_module(module)  # type: ignore
 [ 41](#L41)	  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
 [ 42](#L42)	  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
 [ 43](#L43)	  File "/code/app/db/migrations/env.py", line 82, in <module>
 [ 44](#L44)	    run_migrations_online()
 [ 45](#L45)	  File "/code/app/db/migrations/env.py", line 76, in run_migrations_online
 [ 46](#L46)	    context.run_migrations()
 [ 47](#L47)	  File "<string>", line 8, in run_migrations
 [ 48](#L48)	  File "/usr/local/lib/python3.10/site-packages/alembic/runtime/environment.py", line 853, in run_migrations
 [ 49](#L49)	    self.get_context().run_migrations(**kw)
 [ 50](#L50)	  File "/usr/local/lib/python3.10/site-packages/alembic/runtime/migration.py", line 623, in run_migrations
 [ 51](#L51)	    step.migration_fn(**kw)
 [ 52](#L52)	  File "/code/app/db/migrations/versions/d02dcfbf1517_add_userusageresetlogs_model_and_data_.py", line 30, in upgrade
 [ 53](#L53)	    op.add_column('users', sa.Column('data_limit_reset_strategy', sa.Enum("no_reset", "day","week", "month", "year", name="userdatalimitresetstrategy"), nullable=False, server_default="no_reset"))
 [ 54](#L54)	  File "<string>", line 8, in add_column
 [ 55](#L55)	  File "<string>", line 3, in add_column
 [ 56](#L56)	  File "/usr/local/lib/python3.10/site-packages/alembic/operations/ops.py", line 2047, in add_column
 [ 57](#L57)	    return operations.invoke(op)
 [ 58](#L58)	  File "/usr/local/lib/python3.10/site-packages/alembic/operations/base.py", line 399, in invoke
 [ 59](#L59)	    return fn(self, operation)
 [ 60](#L60)	  File "/usr/local/lib/python3.10/site-packages/alembic/operations/toimpl.py", line 154, in add_column
 [ 61](#L61)	    operations.impl.add_column(table_name, column, schema=schema, **kw)
 [ 62](#L62)	  File "/usr/local/lib/python3.10/site-packages/alembic/ddl/impl.py", line 322, in add_column
 [ 63](#L63)	    self._exec(base.AddColumn(table_name, column, schema=schema))
 [ 64](#L64)	  File "/usr/local/lib/python3.10/site-packages/alembic/ddl/impl.py", line 195, in _exec
 [ 65](#L65)	    return conn.execute(construct, multiparams)
 [ 66](#L66)	  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1380, in execute
 [ 67](#L67)	    return meth(self, multiparams, params, _EMPTY_EXECUTION_OPTS)
 [ 68](#L68)	  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/sql/ddl.py", line 80, in _execute_on_connection
 [ 69](#L69)	    return connection._execute_ddl(
 [ 70](#L70)	  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1472, in _execute_ddl
 [ 71](#L71)	    ret = self._execute_context(
 [ 72](#L72)	  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1943, in _execute_context
 [ 73](#L73)	    self._handle_dbapi_exception(
 [ 74](#L74)	  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 2124, in _handle_dbapi_exception
 [ 75](#L75)	    util.raise_(
 [ 76](#L76)	  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/util/compat.py", line 210, in raise_
 [ 77](#L77)	    raise exception
 [ 78](#L78)	  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1900, in _execute_context
 [ 79](#L79)	    self.dialect.do_execute(
 [ 80](#L80)	  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/engine/default.py", line 736, in do_execute
 [ 81](#L81)	    cursor.execute(statement, parameters)
 [ 82](#L82)	sqlalchemy.exc.ProgrammingError: (psycopg2.errors.UndefinedObject) type "userdatalimitresetstrategy" does not exist
 [ 83](#L83)	LINE 1: ... TABLE users ADD COLUMN data_limit_reset_strategy userdatali...
 [ 84](#L84)	                                                             ^
 [ 85](#L85)	
 [ 86](#L86)	[SQL: ALTER TABLE users ADD COLUMN data_limit_reset_strategy userdatalimitresetstrategy DEFAULT 'no_reset' NOT NULL]
 [ 87](#L87)	(Background on this error at: https://sqlalche.me/e/14/f405)
</code>
 </details>
 
Hi! This PR fixes some pesky errors we ran into during the database(postgres) migration process. 
Basically, certain database types were missing, causing the migration to hiccup.

Updated a couple of existing migration scripts:
- 51e941ed9018_deactive_user_status.py
- 7cbe9d91ac11_proxyhost_security_added.py
- c106bb40c861_alpn_fingerprint_hosts.py
- d02dcfbf1517_add_userusageresetlogs_model_and_data_.py

### CODE REVIEW

**Review by iambabyninja — APPROVED** [review]
👍

**Comment by SaintShit** [comment]
Merged it to `dev`
